README.md

```diff
@@ -1,5 +1,36 @@
-# asanaban
-Kanban on Asana
+# [Asanaban](https://asanaban.com/#!/setup)
+
+![Alt text](app/assets/images/screenshot.png?raw=true "Asanaban Demo")
+## Kanban on Asana
+
+Additional features beyond the Asana Kanban tool include:
+
+- Color-coded tasks
+- Limit to number of tasks per column before column turns red to warn you of too many tasks
+- Shows how many tasks are in a column
+- Multiple "Swimlanes" (rows) to sort tasks
+- Collapsible Columns
+- A variety of awesome background images to choose from 😎
+
+## Connection to Asana for the first time
+
+The first time you go to Asanaban, you'll be forwarded to a [setup page](https://asanaban.com/#!/setup) where you will be walked through a series of steps to connect your Asana account to Asanaban.
```

Comment:
Probably should add something like "This information is stored in local storage, so if you clear it out, the prompt will come back" (or something like that)